Dropped the .spec hunk.

commit 3829b09801daf6af964431db9d3d025f0aff4361
Author: Nalin Dahyabhai <nalin@dahyabhai.net>
Date:   Tue Apr 22 11:01:16 2014 -0400

    Stop explicitly setting a umask for mkhomedir
    
    - stop overriding the system-wide UMASK default in our default
      oddjobd-mkhomedir.conf file (#995097)

diff --git a/src/oddjobd-mkhomedir.conf.in b/src/oddjobd-mkhomedir.conf.in
index b16c7a6..db8bf4e 100644
--- a/src/oddjobd-mkhomedir.conf.in
+++ b/src/oddjobd-mkhomedir.conf.in
@@ -25,14 +25,14 @@
       <interface name="@NAMESPACE@.oddjob_mkhomedir">
 
         <method name="mkmyhomedir">
-          <helper exec="@mypkglibexecdir@/mkhomedir -u 0002"
+          <helper exec="@mypkglibexecdir@/mkhomedir"
                   arguments="0"
                   prepend_user_name="yes"/>
           <!-- no acl entries -> not allowed for anyone -->
         </method>
 
         <method name="mkhomedirfor">
-          <helper exec="@mypkglibexecdir@/mkhomedir -u 0002"
+          <helper exec="@mypkglibexecdir@/mkhomedir"
                   arguments="1"/>
           <allow user="root"/>
         </method>
